### PR TITLE
Fix location of cache.db

### PR DESCRIPTION
**Pull request by persiaAziz**
This should fix the au check fails

### CODE REVIEW

**Comment by persiaAziz** [comment]
@dragon512 please review

**Comment by atsci** [comment]
clang format *successful*! https://ci.trafficserver.apache.org/job/clang-format-github/171/

**Comment by atsci** [comment]
RAT check *successful*! https://ci.trafficserver.apache.org/job/RAT-github/186/

**Comment by atsci** [comment]
FreeBSD11 build *successful*! https://ci.trafficserver.apache.org/job/freebsd-github/1867/

**Comment by atsci** [comment]
AU check *failed*! https://ci.trafficserver.apache.org/job/autest-github/167/

**Comment by atsci** [comment]
Intel CC build *successful*! https://ci.trafficserver.apache.org/job/icc-github/297/

**Comment by atsci** [comment]
Linux build *successful*! https://ci.trafficserver.apache.org/job/linux-github/1757/

File: tests/gold_tests/autest-site/trafficserver.test.ext [hunk 1]
```diff
@@ -120,7 +120,7 @@ def MakeATSProcess(obj, name, command='traffic_server', select_ports=True):
     p.Env['PROXY_CONFIG_HOSTDB_STORAGE_PATH'] = runtime_dir
     # p.Variables.RUNTIMEDIR=runtime_dir
     p.Setup.MakeDir(runtime_dir)
-    p.Setup.Chown(runtime_dir, "nobody", "nobody",ignore=True) 
+    p.Setup.Chown(runtime_dir, "nobody", "nobody",ignore=True)
```
Comment (SolidWallOfCode) [inline tests/gold_tests/autest-site/trafficserver.test.ext]: Do you still need this? I thought the point was to make the chown more specific, as in the code section for the cache.db directory just below.

Comment (persiaAziz) [inline tests/gold_tests/autest-site/trafficserver.test.ext]: I have put the cache.db directory  under runtime_dir, doesn't that mean we still need this?

**Comment by atsci** [comment]
clang-analyzer build *successful*! https://ci.trafficserver.apache.org/job/clang-analyzer-github/430/

File: tests/gold_tests/autest-site/trafficserver.test.ext (hunk 1 above)
Comment (SolidWallOfCode) [inline tests/gold_tests/autest-site/trafficserver.test.ext]: According to the config change, it should be under `runtime_dir/trafficserver`, as indicated by `storage_dir = os.path.join(runtime_dir, 'storage')` and line 140.

**Comment by atsci** [comment]
clang format *successful*! https://ci.trafficserver.apache.org/job/clang-format-github/174/

**Comment by atsci** [comment]
RAT check *successful*! https://ci.trafficserver.apache.org/job/RAT-github/189/

**Comment by atsci** [comment]
AU check *failed*! https://ci.trafficserver.apache.org/job/autest-github/170/

**Comment by atsci** [comment]
FreeBSD11 build *successful*! https://ci.trafficserver.apache.org/job/freebsd-github/1870/

**Comment by atsci** [comment]
Linux build *successful*! https://ci.trafficserver.apache.org/job/linux-github/1760/

**Comment by atsci** [comment]
Intel CC build *successful*! https://ci.trafficserver.apache.org/job/icc-github/300/

**Comment by atsci** [comment]
clang-analyzer build *successful*! https://ci.trafficserver.apache.org/job/clang-analyzer-github/433/

**Comment by atsci** [comment]
clang format *successful*! https://ci.trafficserver.apache.org/job/clang-format-github/179/

**Comment by atsci** [comment]
RAT check *successful*! https://ci.trafficserver.apache.org/job/RAT-github/194/

**Comment by atsci** [comment]
FreeBSD11 build *successful*! https://ci.trafficserver.apache.org/job/freebsd-github/1875/

**Comment by atsci** [comment]
AU check *failed*! https://ci.trafficserver.apache.org/job/autest-github/175/

**Comment by atsci** [comment]
Linux build *successful*! https://ci.trafficserver.apache.org/job/linux-github/1765/

**Comment by atsci** [comment]
clang-analyzer build *successful*! https://ci.trafficserver.apache.org/job/clang-analyzer-github/438/

**Pull request merged**